images-list

```diff
@@ -131,6 +133,8 @@ ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operat
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.1
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.3
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.4
+ghcr.io/epinio/epinio-server rancher/mirrored-epinio-server 0.7.1
```

Comment:
Wrong repo created here, found https://hub.docker.com/r/rancher/mirrored-epinio-server but should be:

```suggestion
ghcr.io/epinio/epinio-server rancher/mirrored-epinio-epinio-server 0.7.1
```

Reply:
I am quite confused here.

The line in the images-list and the repo you found look to be matching to me (see below, lines 1 and 2, aligned vertically).
Your suggested change then makes them different (lines 2, 3).

Can you explain in a bit more detail ?

```
1. ghcr.io/epinio/epinio-server rancher/mirrored-epinio-server
2.     https://hub.docker.com/r/rancher/mirrored-epinio-server
3. ghcr.io/epinio/epinio-server rancher/mirrored-epinio-epinio-server
```

Reply:
@aiyengar2 :arrow_up:

Reply:
Resolved per meeting. Commit 1d0a124a257814e3f8c9de4780679e387989994f